### ISSUE TITLE
[Enhancement]: aws_amplify_domain_association certificate_settings AMPLIFY_MANAGED should be default

### ISSUE THREAD

**Issue by rsi-mrobinson** [hashicorp/terraform-provider-aws#38387]
### Description

the newly added (>=5.57.0) certificate_settings attribute should default to AMPLIFY_MANAGED rather than null. Existing resources appear to result in plans where the certificate settings are being removed, despite amplify managed domains being the default up to this point. The new certificate setting block should respect this an only be required to override the default settings.

### Affected Resource(s) and/or Data Source(s)

aws_amplify_domain_association

### Would you like to implement a fix?

None

**Comment by justinretzolk** [comment]
Hey @rsi-mrobinson 👋 Thank you for taking the time to raise this! To make sure I understand -- you're saying that if you update the version and make no other configuration changes, resources are being replaced on the next apply?

**Comment by rsi-mrobinson** [comment]
That's correct, if you upgrade to 5.57 or above without making changes to this resource, the plan shows a change to remove the default amplify managed dns

**Comment by aayushsss1** [comment]
Hey @justinretzolk @rsi-mrobinson  I can take this issue up, just to be clear the only change would be to add 
`Default : "AMPLIFY_MANAGED"` here right? 

https://github.com/hashicorp/terraform-provider-aws/blob/c3b9b48e3b84197374aa8b3263c2b12ded3acfcc/internal/service/amplify/domain_association.go#L61-L67

**Comment by rsi-mrobinson** [comment]
That's what I'm thinking
